values.yaml

```diff
@@ -1331,7 +1341,8 @@ syncCatalog:
       cpu: "50m"
 
   # Log verbosity level. One of "trace", "debug", "info", "warn", or "error".
```

Comment:
Should we say something like "Override `global.logLevel`" for all of these overrides?